SelcouthDigital/react-native-action-sheet
# [GENERAL] Modal component removed + custom styles handler added || Refactor/modal

# Summary

- Removes the Modal component which apparently is going to be deprecated from the React Native core.
- It was also causing dismiss issues on specific Android devices.
- Adds a custom cancel button prop
- Adds custom styles handling
- Code clean up

## Screens/Gifs

TBC

## Type of change


## Severity of change

- [x] Breaking change (major update) 🚨

## Steps to test

1. Check out the branch
2. Run the app and Android and iOS
3. Notice behaviour

## Your checklist for this pull request

🚨Please review the guidelines for contributing to this repository.

- [x] Make sure you are making a pull request against the canary branch (left side). Also you should start your branch off our canary.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have tested code on both Android and iOS devices

💔Thank you!
